Content.Server/StationEvents/Events/IonStormRule.cs

```diff
@@ -206,21 +203,20 @@ private string GenerateLaw()
             _ => Loc.GetString("ion-storm-heads")
         };
         var part = Loc.GetString("ion-storm-part", ("part", RobustRandom.Prob(0.5f)));
-        var harm = RobustRandom.Next(0, 7) switch
+        var harm = RobustRandom.Next(0, 6) switch
         {
             0 => concept,
             1 => $"{adjective} {threats}",
             2 => $"{adjective} {objects}",
             3 => Loc.GetString("ion-storm-adjective-things", ("adjective", adjective)),
-            4 => species,
-            5 => crew1,
+            4 => crew1,
             _ => Loc.GetString("ion-storm-x-and-y", ("x", crew1), ("y", crew2))
         };
 
         if (plural) feeling = feelingPlural;
 
         // message logic!!!
-        return RobustRandom.Next(0, 37) switch
+        return RobustRandom.Next(0, 36) switch
         {
```

Comment:
Which psycho coded this to not just use a string array and length?

Reply:
delta

Reply:
me

Reply:
it's dm so I'd believe it.